src/api/app/assets/javascripts/webui2/users_groups.js

```diff
@@ -1,48 +1,61 @@
+function initializeUserConfigurationDatatable() { // jshint ignore:line
+  projectsDatatable(
```

Comment:
`projectsDatatable` seems to be generic enough to be reused, so we could rename it to `initializeServerSideDatatable` and move it under [app/assets/javascripts/webui2/datatables.js](https://github.com/openSUSE/open-build-service/blob/ed035576b5e8421c6a4583b9ea919601d36489ac/src/api/app/assets/javascripts/webui2/datatables.js)

Reply:
True that. I'll change it.

Reply:
@dmarcoux updated please have another look